configGlobal.go

```diff
@@ -125,24 +133,84 @@ func setGlobalConfigVariables(storageDir string, cfgFile string) error {
 		globalLogFileCount = configInt
 	}
 
-	// No form of defaults for E-Mail settings, just read them in
-	emailFromAddress = viper.GetString("emailFromAddress")
-	emailToAddress = viper.GetString("emailToAddress")
-	emailServerHostname = viper.GetString("emailServerHostname")
-	emailServerPort = viper.GetInt("emailServerPort")
-	emailAuthUsername = viper.GetString("emailAuthUsername")
-	emailAuthPassword = viper.GetString("emailAuthPassword")
+	// Support for deprecated -m flag until it gets removed
+	if sendMail {
```

Comment:
Should this be done if email notifications were specified in the configuration file? It could be hard to figure out (I guess you'd need to parse notifiers and then, if no email notifier was found && sendMail is set, add them. But not sure if it's worth it given that it will eventually go away anyway.

What are your thoughts?

Reply:
I think it's ok to leave it as is. This fulfils the contract of -m flag and like you say it will go away eventually.